graphs/uniform_search_cost.py

```diff
@@ -0,0 +1,260 @@
+# diagonal clockwise
+dxy1 = [
```

Comment:
Provide a wikipedia or some page through which a user can understrand what you are trying to implement, define it briefly if you can.